quantum/process_keycode/process_space_cadet.c

```diff
@@ -97,7 +93,12 @@ void perform_space_cadet(keyrecord_t *record, uint16_t sc_keycode, uint8_t holdM
             register_mods(MOD_BIT(holdMod));
         }
     } else {
-        if (sc_last == holdMod && timer_elapsed(sc_timer) < get_tapping_term(sc_keycode, record)) {
+#ifdef TAPPING_TERM_PER_KEY
```

Comment:
I was under the impression that `get_tapping_term()` simply returned `TAPPING_TERM` by default? That's what it looks like in action_tapping.c.

Reply:
It does, but in theory, shouldn't call the function unless the per key define exists, to keep the behavior consistent.